packages/core/src/components/button/button.scss

```diff
@@ -1,3 +1,209 @@
 :host {
-  display: block;
+  display: inline-block;
+  width: auto;
+  cursor: pointer;
+}
+
+.button {
+  box-sizing: border-box;
```

Comment:
I don't think we should be setting this property. It's typically set at the document level

Reply:
Yeah oops I think that was left there from my testing and I forgot to take it out